packages/element-templates-json-schema-shared/src/defs/examples.json

```diff
@@ -35,5 +35,15 @@
         "notEmpty": true
       }
     ]
+  },
+  "elementType": {
+    "examples": [
+      "bpmn:Task",
+      "bpmn:ServiceTask",
+      "bpmn:SequenceFlow",
```

Comment:
Should we support the use case of morphing into sequenceFlow? I think we should take this out of scope and only say we move 
* taskLike <-> taskLike
* taskLike <-> gatewayLike (maybe)?

Reply:
I agree that we should only allow morphing types that are compatible (Task <-> Task, Gateway <-> Gateway and Sequence Flow <-> Sequence Flow). This is something we can't verify in a schema. I think we should keep it in the examples to show that it is not limited to Tasks only

Reply:
For context, examples are only editor-related and don't provide validation (https://github.com/camunda/element-templates-json-schema/pull/45).

Reply:
But would the cases

* Sequence Flow like <-> Sequence Flow like
* Process like <-> Process like 

be supported? What would be a concrete use case for that? If not, I would not add examples for these.

Reply:
Slightly related context: we even considered to not support `bpmn:Process` for elementTemplates entirely since there is at least one open bug which is hard to fix. See: https://github.com/camunda/camunda-modeler/issues/2128

Reply:
Or just add some examples which would actually make sense in a real world use case? So e.g. we could mention `bpmn:ServiceTask, bpmn:UserTask, bpmn:SendTask, ...`

Reply:
I added a few more specific Task and Gateway Types to the examples and removed the generic ones: [dea7727](https://github.com/camunda/element-templates-json-schema/pull/48/commits/dea7727a34dbb0ee31db82702fb8d262f91784f4)